bundles/EcommerceFrameworkBundle/FilterService/FilterType/ElasticSearch/MultiSelectRelation.php

```diff
@@ -23,8 +23,10 @@ class MultiSelectRelation extends \Pimcore\Bundle\EcommerceFrameworkBundle\Filte
 {
     public function prepareGroupByValues(AbstractFilterDefinitionType $filterDefinition, ProductListInterface $productList)
     {
-        $field = $this->getField($filterDefinition);
-        $productList->prepareGroupByRelationValues($field, true, !$filterDefinition->getUseAndCondition());
+        if (method_exists($filterDefinition, 'getUseAndCondition')) {
```

Comment:
besides that, we should check `$filterDefinition instanceof \Pimcore\Model\DataObject\Fieldcollection\Data\FilterMultiRelation`, other wise throw `InvalidConfigException`

Reply:
still open